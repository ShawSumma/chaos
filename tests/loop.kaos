list a = [
    1,
    2,
    3
]

20 times do:
    print a
end

num b = 32
print b
