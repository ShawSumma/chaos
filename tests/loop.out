[1, 2, 3]
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
[1, 2, 3]
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
[1, 2, 3]
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
256
1
deepest loop
1
deepest loop
after all the loops
1
5
2
5
3
5
[5, 5, 5]
